### PR TITLE
Inicial implementation of Profile Page

### PR DESCRIPTION
**Pull request by rafaelchimicatti**
Implementação inicial de Página de Perfil.

### CODE REVIEW

File: frontend/src/pages/Profile/Profile.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  .profile-container {
+    display: flex;
+    margin-left: 20vw;
+    width: calc(100vw - 20vw); 
+    height: 100vh;
+    overflow-x: hidden;
+    overflow-y: scroll; 
+  }
+  
+  .profile-screen {
```
Comment (beviggiano) [inline frontend/src/pages/Profile/Profile.css:17]: Consider removing the following padding properties if not reused in the future

**Review by beviggiano — APPROVED** [review]

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -22,7 +23,9 @@ export const App = () => {
                     <Route path="/login" element={<PublicRoute element={<Login />} />} />
                     <Route path="/signup" element={<PublicRoute element={<SignUp />} />} />
                     {/*<Route path="/posts" element={<Posts/>} />*/}
-                    <Route path="/posts" element={<PrivateRoute element={<AdvertisementScreen />} />} />
+                    <Route path="/posts" element={<PublicRoute element={<AdvertisementScreen />} />} />
+                    
+                    <Route path="/profile" element={<PublicRoute element={<ProfileScreen />} />} />
```
Comment (edu-correia) [inline frontend/src/App.tsx:28]: These 2 pages should use PrivateRoute, right?
In other words, these should be blocked to access if the user is not authenticated

**Review by edu-correia — COMMENTED** [review]
Just one question

**Review by juancoelhoo — APPROVED** [review]

File: frontend/src/App.tsx (hunk 2 above)
Comment (rafaelchimicatti) [reply frontend/src/App.tsx:28]: Yes! I was using it only for debbuging and forgot to revert these changes. Will be fixed.

File: frontend/src/pages/Profile/Profile.css (hunk 1 above)
Comment (rafaelchimicatti) [reply frontend/src/pages/Profile/Profile.css:17]: Ok!

**Pull request merged**